src/platform/nrfconnect/wifi/NrfWiFiDriver.cpp

```diff
@@ -147,6 +147,11 @@ CHIP_ERROR NrfWiFiDriver::CommitConfiguration()
 
 CHIP_ERROR NrfWiFiDriver::RevertConfiguration()
 {
+    if (WiFiManager::StationStatus::CONNECTED <= WiFiManager::Instance().GetStationStatus())
+    {
+        WiFiManager::Instance().Disconnect();
```

Comment:
What if the device is not yet connected but is connecting, i.e. wrong SSID has been provided? We should stop the connection establishment, right?

Reply:
Ok, I checked if the disconnect request is sane when the supplicant is during connection establishment - and it should work fine, I will fix it.

Reply:
Done